### PR TITLE
Update after first evaluation round

### PR DESCRIPTION
**Pull request by nuest**
- [x] update according to user survey
- [x] check support chat
- [x] merge #22 
- [x] export rich diff

### CODE REVIEW

**Comment by nuest** [comment]
Created a diff with

```bash
git diff f9b2f618a61392cd0f7cbc5200d003fa426a3790 705c659ff5138cedc2b2b1615e57d22293d43e4e > eval_1.diff
```

and stored it in the evaluation documents directory along with PDFs of the GitHub diff view.

**Comment by nuest** [comment]
@MarkusKonk @7048730 please check the diff docs and let me know if they are sufficient in your opinion. Then we can merge.

**Comment by ghost** [comment]
looks good to me 👍

File: docs/dev-guide/index.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ One of the biggest issues is the **scope of licenses**, namely what to do about
 - better user experience (otherwise all info must be transported via stdout)
 - to be sure nothing is manipulated within the validation script
 
+### Why is the data not in the image (inner contaienr) but in the outer container
```
Comment (Timmimim) [inline docs/dev-guide/index.md:72]: Not to be nitpicky, but there is a typo in the word container (inside the brackets).

Comment (nuest) [reply docs/dev-guide/index.md:72]: Nitpickiness is good. PR with a fix is welcome.

**Pull request merged**